controllers/machineset_controller_test.go

```diff
@@ -562,7 +563,7 @@ var _ = Describe("MachinesetController", func() {
 			It("should delete taint in node", func() {
 				err = r.updateTaintsInNode(ctx, &machine, &node)
 				Expect(err).NotTo(HaveOccurred())
-				Expect(node.Spec.Taints).To(Equal(updatedNode.Spec.Taints))
+				Expect(updatedNode.Spec.Taints).To(Equal(newTaintsInMachine))
```

Comment:
Even though they are the same values here, shouldn't this be `.To(Equal(updatedTaintsInNode))` since we're checking for the node update here?